build/tasks/release.js

```diff
@@ -6,33 +6,18 @@ var $ = require('gulp-load-plugins')({
 var paths = require('../paths');
 
 gulp.task('cache-bust', function () {
-  var cacheBust = "var systemLocate = System.locate; System.locate = function(load) { var cacheBust = '?bust=' + " + Math.round(new Date() / 1000) +"; return Promise.resolve(systemLocate.call(this, load)).then(function(address) { if (address.indexOf('bust') > -1 || address.indexOf('css') > -1 || address.indexOf('json') > -1) return address; return address + cacheBust; });}\n"
   return gulp.src('dist/app/app.js')
     .pipe($.insert.prepend("window.prod = true;\n"))
-    .pipe($.insert.prepend(cacheBust))
```

Comment:
The `window.prod` isn't used anywhere and should be removed. I'm using `angular-environment` instead. If you want to remove the cache-bust stuff, this whole task should be removed.